src/APNSClient.php

```diff
@@ -3,38 +3,24 @@
 
 class APNSClient {
 
-	/** @var resource */
-	private $curl_handle;
+	/** @var APNSNetworkService **/
+	private $network_service;
 
 	/** @var APNSConfiguration */
 	private $configuration;
 
 	/** @var string */
 	private $provider_token;
 
-	/** @var int */
-	private $port_number = 443;
-
-	/** @var bool */
-	private $debug = false;
-
-	/** @var bool */
-	private $disable_ssl_verification = false;
```

Comment:
Because we decided to add those properties to `APNSNetworkService`, I removed them from the client itself and update the setter methods to pass the information through to set the corresponding value in the network layer.